### ISSUE TITLE
HTML Tags in scripts break editor, and/or are lost on save/re-edit.  

### ISSUE THREAD

**Issue by jcfergus** [jsreport/jsreport#146]
There appears to be an issue in the scripts extension with HTML tags contained within the code - most likely an escaping issue.  If you put a line in your script, i.e. 

```
var stuff = "<html><head><title>blah</title></head><body>stuff</body></html>";
```

then save your script and go back to re-edit it, the tags will all be missing.   (They are in the saved code, but disappear if you save again.) I've also run into situations where not only are the tags missing, but if one of those tags happens to be </div> it can cause the editor to be non-functional until you go edit the saved code to remove the tags and restart the server.  

I'm sure this is just an escaping issue but haven't dug into how the Ace editor handles that enough to supply a patch.

**Comment by bjrmatos** [comment]
you're right, I've been able to reproduce it, let us see if we can figure it out what happens

**Comment by pofider** [comment]
Fixed in this commit
https://github.com/jsreport/jsreport-scripts/commit/bf87c864fd1d3984bace510e59c091e41c950f42

It will be included in v 0.10 release. It should be published today.
